KlingonAssistant/data/mem-17-tlh.xml

```diff
@@ -804,17 +804,17 @@ There is an idiom, {Hem; tlhIngan rur}.</column>
       <column name="_id">33701</column>
       <column name="entry_name">tlhIngan Hol</column>
       <column name="part_of_speech">n:deriv</column>
-      <column name="definition">Klingon (language)</column>
+      <column name="definition">Klingon language</column>
```

Comment:
Alright, thnaks for the notice. I may have been distracted from the phrase in KGT that says "literally Klingon language". But okrand continues says "henceforth called Klingon".